src/ucs/config/parser.c

```diff
@@ -1139,3 +1139,18 @@ size_t ucs_config_memunits_get(size_t config_size, size_t auto_size,
         return ucs_min(config_size, max_size);
     }
 }
+
+int ucs_config_names_search(ucs_config_names_array_t config_names,
+                            const char *str)
```

Comment:
the list of patterns is in config_names array. the str is one of the counters names